app/styleguides/_default/components/subheader.css

```diff
@@ -6,3 +6,43 @@ $helvetica-stack: Helvetica, sans-serif;
   font: bold 22px $helvetica-stack;
   margin: 0 0 1em;
 }
+
+.ui-radio-group.is-vertical .ui-radio-group__radios {
```

Comment:
Are these styles for edit mode? It's better to just use Kiln's default styles for things rather than trying to override.